resources/idf-python-wheels_diagram_source.txt

```diff
@@ -0,0 +1,79 @@
+@startuml
+
+:build-wheels-platforms.yml;
+start
+
+:Install OS dependencies (os_dependencies directory);
+
+:Build wheels for IDF (build_wheels.py)
+
+- minimum IDF-supported Python version;
+start
+
+:Fetch all ESP-IDF branches;
+
+:Get used ESP-IDF branches (specified from GitHub variables);
+
+:Get constraints files for used ESP-IDF branches;
+
+:                      Assemble requirements
+- create requirements.txt
+- transform yaml to requirement from exclude_list.yaml
```

Comment:
I think it might be clearer instead of no. 2 & 3: "remove requirements defined by exclude_list.yaml"